sklearn/feature_selection/_from_model.py

```diff
@@ -283,5 +284,7 @@ def n_features_in_(self):
         return self.estimator_.n_features_in_
 
     def _more_tags(self):
-        estimator_tags = self.estimator._get_tags()
-        return {'allow_nan': estimator_tags.get('allow_nan', True)}
+        return {
+            'allow_nan':
+            _safe_tags(self.estimator, key="allow_nan", default=True)
```

Comment:
I guess this isn't stricly from this PR but what's the reasoning for `default=True`? We want to be as permissive as possible?

Reply:
I assume that it was the intent in https://github.com/scikit-learn/scikit-learn/issues/15319